recipes/transannot/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "3-70b2a60" %}
-{% set sha256 = "3b1d2f28362004378528755b8b4e0b8dc45b51cc60562816258bdf16d9cda08d" %}
+{% set version = "3-e15e316" %}
```

Comment:
I don't think this PR will fix the issue.
```
 version: {{ version|replace("-", ".")  }}
```
will produce `3.e15e316` but the next hash could be something like `[0-9].....` and since the digits are before letters (`e`), so the new version will look like an older version.